website/blog/2022-01-11-aluminium.md

```diff
@@ -0,0 +1,585 @@
+---
+author: Tomasz Godzik
+title: Metals v0.11.0 - Aluminium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.11.0! Due to the number of
+changes in this release we decided to introduce one more minor version before
+Metals 1.0.0, which is planned to come this year.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">313</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">129</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">11</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">10</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/<num>?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Add CFR class file viewer.
+- [Scala 3] Show type on hover.
+- Simplify usage of sbt server.
+- Better support for mill BSP.
+- Extract value code action.
+- Add command to run current file.
+- Support test explorer API.
+- Add go to definition via links for synthetic decorations.
+- Basic Java support.
+- Add support for Scala 2.13.8
+
+## Add CFR class file viewer
+
+TODO
+
+## [Scala 3] Show type on hover
+
+TODO
+
+## Simplify usage of sbt server
+
+TODO
```

Comment:
Previously if you wanted to use sbt as your build server instead of the default Bloop you'd need to ensure a `.bsp/sbt.json` file existed _before_ you tried to switch your build server. We originally created the `metals.generate-bsp` command for this, but realized this potentially forced extra steps on the user. To make this simpler, now even if no BSP entry exists for sbt and you attempt to switch your build server to sbt via the `metals.bsp-switch` command, we'll either use the existing `.bsp/sbt.json` file if one exists or automatically generate it (needs at least sbt 1.4.1) for you and then switch to it.

Reply:
Thanks!